assets/src/setup/pages/save/index.js

```diff
@@ -3,25 +3,136 @@
+				<path d="M10.4722 81.5586L12.8722 88.1586L19.4722 90.5586L12.8722 92.9586L10.4722 99.5586L8.07217 92.9586L1.47217 90.5586L8.07217 88.1586L10.4722 81.5586Z" fill="white" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" strokeLinecap="round" strokeLinejoin="round" />
+				<path d="M144.095 75.0624L145.303 84.4721L151.979 86.8034L156.709 93.8481L155.359 100.717L164.592 105.589L169.674 100.473L178.103 99.0727L184.393 103.366L191.437 98.6364L189.651 92.1705L194.524 82.9369L201.182 80.1693L199.782 71.7409L193.106 69.4095L187.395 62.172L188.745 55.3034L180.493 50.6238L175.218 56.7209L166.982 57.1403L159.711 52.6535L153.84 56.5953L155.434 64.0425L150.561 73.276L144.095 75.0624Z" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" />
+				<path d="M170.614 87.9204C176.033 88.9857 181.29 85.4562 182.355 80.037C183.42 74.6179 179.891 69.3612 174.471 68.296C169.052 67.2307 163.796 70.7603 162.73 76.1794C161.665 81.5986 165.195 86.8552 170.614 87.9204Z" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" />
+				<path d="M138.394 47.0394L148.416 53.086L146.101 64.8607L134.536 66.6639L131.609 71.1842L134.394 82.9421L123.827 90.0372L113.998 83.0094L108.706 84.0074L103.834 93.241L91.0776 90.7336L90.0628 80.3427L84.7541 76.2417L74.9587 79.4119L67.8636 68.845L73.7173 59.8043L72.7192 54.5124L62.6973 48.4659L65.0119 36.6912L76.5768 34.888L79.6966 29.3864L76.7193 18.6098L87.0933 12.4959L96.9223 19.5237L102.407 17.5445L107.28 8.31088L120.035 10.8183L122.032 21.4021L126.359 25.3101L137.136 22.3328L144.424 31.9185L137.589 40.7664L138.394 47.0394Z" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" />
+				<path d="M103.628 60.5884C109.047 61.6537 114.304 58.1241 115.369 52.705C116.434 47.2858 112.905 42.0292 107.486 40.964C102.066 39.8987 96.8098 43.4282 95.7445 48.8474C94.6793 54.2665 98.2088 59.5232 103.628 60.5884Z" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" />
+				<path d="M101.718 70.3011C112.502 72.421 122.963 65.3972 125.083 54.6131C127.203 43.829 120.179 33.3683 109.395 31.2485C98.6109 29.1286 88.1502 36.1524 86.0303 46.9365C83.9105 57.7206 90.9342 68.1813 101.718 70.3011Z" stroke="#2459E7" strokeWidth="2" strokeMiterlimit="10" />
+			</svg>
+
+			<h1>
+				{ __( 'Saving your settings …', 'amp' ) }
+			</h1>
+		</div>
+	);
+}
+
+function Preview() {
+	const {
+		editedOptions: { theme_support: themeSupport },
+		originalOptions: { preview_permalink: previewPermalink },
+	} = useContext( Options );
+
+	return (
+		<>
+			<AMPInfo icon={ ( props ) => <IconMobile { ...props } /> }>
+				{ 'standard' === themeSupport
+					? __( 'Live view of your site', 'amp' )
+					: __( 'Live view of a page on your site using AMP mode', 'amp' )
+				}
+			</AMPInfo>
+			<Phone>
+				<iframe
+					className="done__preview-iframe"
+					src={ previewPermalink }
+					title={ __( 'Site preview', 'amp' ) }
+					name="amp-wizard-completion-preview"
```

Comment:
Minor suggestion that I'm not totally sold on myself: since the admin bar gets hidden at `DOMContentLoaded` there is a flash of the admin bar. This could be avoided if we hide the `iframe` until the page is loaded.

```suggestion
					name="amp-wizard-completion-preview"
					style={ { opacity: 0 } }
					onLoad={ ( event ) => { event.target.style.opacity = '1'; } }
```

There's probably a more elegant way to do this however. (Including showing a spinner initially.) 

Not important.

Reply:
Done in 9b707af3265b3f67067f45029f5030ef8d2c8221